### PR TITLE
Add content, modify layouts for vitess.io.

### PR DESCRIPTION
**Pull request by AndyDiamondstein**

### CODE REVIEW

File: doc/GettingStartedKubernetes.md [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+
+1.  Log in to the Google Developers Console to [enable billing]
+    (https://console.developers.google.com/billing).
+    1.  Click the **Billing** pane if you are not there already.
+    1.  Click **New billing account**
+    1.  Assign a name to the billing account -- e.g. "Vitess on
+        Kubernetes." Then click **Continue**. You can sign up
+        for the free trial to avoid any charges. However, even if you
+        skip the free trial, you should not come anywhere close to
+        exceeding the [App Engine free quota]
```
Comment (enisoc) [inline doc/GettingStartedKubernetes.md]: Does the page you linked also apply to Compute Engine?

**Comment by alainjobart** [comment]
@yaoshengzhe Please review this, thanks!

File: doc/VitessOverview.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+## Comparisons to other storage options
+
+The following sections compare Vitess to two common alternatives, a vanilla MySQL implementation and a NoSQL implementation.
+
+### Vitess vs. Vanilla MySQL
+
+Vitess improves a vanilla MySQL implementation in several ways:
+
+<style>
+  table td {
```
Comment (enisoc) [inline doc/VitessOverview.md]: It looks like GitHub's markdown renderer doesn't like this. The CSS shows through literally. You can click View in the Pull Request diff to preview the rendered version.

File: vitess.io/_layouts/homebase.liquid [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <h4>Navigation</h4>
+    {% include nav.liquid %}
+  </nav>
+</div>
+
+{{ content }}
+
+<footer>
+  <div class="container">
+    <p class="small">&copy; 2014 Google. Licensed under <a href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>
```
Comment (yaoshengzhe) [inline vitess.io/_layouts/homebase.liquid]: 2015 Google ?

File: vitess.io/_includes/analytics.liquid [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+  if (window.location.hostname == 'youtube.github.io') {
+    ga('create', '{{ site.analytics.id }}', 'auto');
```
Comment (yaoshengzhe) [inline vitess.io/_includes/analytics.liquid]: you didn't set analytics.id in _config.yml. I've enabled google analytics for the site already and you may want to use id "UA-60219601-1" in the config file.

File: vitess.io/_config.yml [hunk 5]
```diff
@@ -34,4 +34,4 @@ owner:
   disqus-shortname:
```
Comment (yaoshengzhe) [inline vitess.io/_config.yml]: set analytics.id to "UA-60219601-1"

File: doc/VitessOverview.md (hunk 2 above)
Comment (AndyDiamondstein) [inline doc/VitessOverview.md]: Thanks for noticing this. I removed the CSS from this file and stuck it in the extra.css file.

**Comment by yaoshengzhe** [comment]
Please also do a local rebase first otherwise merge will fail as indicated by github:
In your own local feature branch, do "git rebase master", you shall see some rebase errors. Please fix it and then update your pull request again.

Errors I got:
First, rewinding head to replay your work on top of it...
Applying: Update diagram showing Vitess components.
Applying: Add content for vitess.io.
Using index info to reconstruct a base tree...
M	vitess.io/_layouts/default.html
M	vitess.io/_layouts/home.html
<stdin>:338: trailing whitespace.
    Kubernetes API server without a need for additional setup.
<stdin>:358: trailing whitespace.
    <br>**Note:** If you do not include the <code>-force</code> flag
<stdin>:622: trailing whitespace.
The diagram below illustrates Vitess' components:
<stdin>:2036: trailing whitespace.
    stream so that a future shard master only gets statements that
<stdin>:5292: trailing whitespace.
          against data storage alternatives.
warning: squelched 2 whitespace errors
warning: 7 lines add whitespace errors.
Falling back to patching base and 3-way merge...
CONFLICT (modify/delete): vitess.io/_layouts/home.html deleted in Add content for vitess.io. and modified in HEAD. Version HEAD of vitess.io/_layouts/home.html left in tree.
CONFLICT (modify/delete): vitess.io/_layouts/default.html deleted in Add content for vitess.io. and modified in HEAD. Version HEAD of vitess.io/_layouts/default.html left in tree.
Failed to merge in the changes.
Patch failed at 0002 Add content for vitess.io.
The copy of the patch that failed is found in:
   /Users/shengzhe/GitHub/vitess/.git/rebase-apply/patch

File: doc/GettingStartedKubernetes.md (hunk 1 above)
Comment (AndyDiamondstein) [inline doc/GettingStartedKubernetes.md]: I removed that sentence and instead linked to the page about the Cloud Platform trial. I think we can go back later on and determine whether there's a free quota for GCE. In the meantime, the free trial page confirms that the credit is for any Cloud Platform services, including GCE. Under the free trial, you're limited to 8 concurrent cores.

File: vitess.io/_includes/analytics.liquid (hunk 4 above)
Comment (AndyDiamondstein) [inline vitess.io/_includes/analytics.liquid]: Added this ID to the _config.yml file ... thanks!

File: vitess.io/_config.yml (hunk 5 above)
Comment (AndyDiamondstein) [inline vitess.io/_config.yml]: Done.

**Comment by yaoshengzhe** [comment]
Had an offline discussion with Andy, resolved merge issue. LGTM

**Pull request merged**